extensions/amp-analytics/0.1/iframe-transport-vendors.js

```diff
@@ -28,3 +28,8 @@ export const IFRAME_TRANSPORTS = /** @type {!JsonObject} */ ({
   'bg': 'https://tpc.googlesyndication.com/b4a/b4a-runner.html',
   'moat': 'https://z.moatads.com/ampanalytics093284/iframe.html',
 });
+
+export const IFRAME_TRANSPORTS_CANARY = /** @type {!JsonObject} */ ({
```

Comment:
worth to save some bytes for the moat duplicates
```js
export const IFRAME_TRANSPORTS_CANARY = Object.assign({}, IFRAME_TRANSPORTS, {bg: 'xxx'});
```

Reply:
Done.